css/abstract/colors.css

```diff
@@ -9,4 +9,8 @@
     --icon-color:grey;
```

Comment:
you have `--grey-color` already you can use that.

Reply:
made the changes thank you